src/envs/cover.py

```diff
@@ -1255,3 +1255,32 @@ def _create_initial_state(self, blocks: List[Object],
         del rng
         zero_rng = np.random.default_rng(0)
         return super()._create_initial_state(blocks, targets, zero_rng)
+
+
+class CoverMultistepOptionsHolding(CoverMultistepOptions):
+    """A variation of CoverMultistepOptions where the goals only involve
+    Holding.
+
+    This environment is useful for debugging option learning.
```

Comment:
this is just because we only have to get picking right, not placing?

Reply:
Yeah right